config.json

```diff
@@ -1306,7 +1292,7 @@
           "recursion",
           "ranges"
         ],
-        "practices": ["charlists", "ranges"],
+        "practices": ["strings", "charlists"],
```

Comment:
I'm not sure if this adds `strings` on purpose or by accident. My suggestion was to make this an `enum` exercise. The example solution doesn't even use a single `String` function, only `Enum.join`.